tests/e2e/webdash_test.go

```diff
@@ -1,4 +1,4 @@
-package tests
+package e2e
```

Comment:
Since all the tests seems to have moved into e2e does it still make sense to have the e2e subdirectory and package?